phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewsConfigurationLibrary.java

```diff
@@ -39,11 +39,8 @@
 
     @Bean
     public ViewLibrary viewLibrary() {
-        return new ViewLibrary(
-                applicationName,
-                themeConfiguration.themeManager(),
-                phoenicisGlobalConfiguration.objectMapper()
-        );
+        return new ViewLibrary(applicationName, themeConfiguration.themeManager(),
```

Comment:
I see. Can we somehow make sure that this always happens? Or at least Jenkins gives an error? What I did once is to run `git status` in the Jenkins build and return an error if something changed.